src/main/java/uk/gov/register/presentation/dao/Entry.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.register.presentation.dao;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.sql.Timestamp;
+import java.text.SimpleDateFormat;
+import java.time.ZoneId;
+import java.util.TimeZone;
+
+public class Entry {
+    private final static SimpleDateFormat simpleDateFormat = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm:ss'Z'");
```

Comment:
is there an existing formatter for this? given we're going for standard RFC3339 I'd expect there to be an existing formatter